bin/helm-deploy

```diff
@@ -11,33 +11,35 @@ CI_REF="${CI_TAG}"
 CI_REF="${CI_REF:-$CI_BRANCH}"
 CI_REF=$(echo "${CI_REF}" | tr / _)
 
-if [[ ! $(kubectl get namespace $NAMESPACE) ]]; then
-  echo "Creating ${NAMESPACE} namespace"
-  # shellcheck disable=SC2086
-  kubectl create namespace $NAMESPACE;
-fi
+if [ "${HELM_TEMPLATE_ONLY}" != "true" ]; then
+  if [[ ! $(kubectl get namespace "$NAMESPACE") ]]; then
+    rok8s_echo "Creating ${NAMESPACE} namespace"
+    # shellcheck disable=SC2086
```

Comment:
Is it actually necessary to deisable SC2086 here?  I feel like this should work fine as `kubectl create namespace "$NAMESPACE";